src/validation/rules/SingleFieldSubscriptionsRule.js

```diff
@@ -6,9 +6,10 @@ import type { OperationDefinitionNode } from '../../language/ast';
 import type { ASTValidationContext } from '../ValidationContext';
 
 /**
- * Subscriptions must only include one field.
+ * Subscriptions must only include a non-introspection field.
```

Comment:
This is very precise wording; if we were to use `one non-introspection field` or `a single non-introspection field` here then it wouldn't place limits on the _introspection_ fields we added, only the non-introspection fields.